output/cloud/expv2/hdr.go

```diff
@@ -151,6 +164,9 @@ func histogramAsProto(h *histogram, time int64) *pbcloud.TrendHdrValue {
 	if h.ExtraHighBucket > 0 {
 		hval.ExtraHighValuesCounter = &h.ExtraHighBucket
 	}
+	// We don't expect to change the minimum resolution at runtime
+	// so a pointer is safe here
+	hval.MinResolution = &h.MinimumResolution
```

Comment:
Is there are reason to use a pointer though?

It won't save memory and will add dereferencing the point when we have to actually write it.

Reply:
I guess the problem is that this is because in the protobuf definition this is not required :facepalm: 

Not certain we can do anythign about this :shrug:

Reply:
Yeah, the reason is how Protobuf defines optional values. We haven't released yet the new cloud output v2 as the default so we could think of migrating directly to a new Protobuf version making it required. But my feeling is that it would require a maintenance window.

Reply:
I believe that if we un-optional this field, this doesn't really change anything for the backend?

Reply:
@esquonk if the suggestion is to change it only on the client then it would require having different proto file versions between the backend and client, that doesn't sound optimal to me.
Instead, if you mean for both of them, then In that case, the problem is how to handle the versioning. If we set the field on the backend as required then the already k6 v0.46.0 released version will stop working if we force the `K6_CLOUD_API_VERSION=2` overwriting it from the backend. We could consider it as a no-problem as we are pretty confident that we will set it directly from the code releasing v0.47.0, but I'm not sure. Thoughts?